src/api/controllers/v2/enrollment.controller.ts

```diff
@@ -58,17 +61,10 @@ export const deleteEnrollmentById = async (
   req: WithAuthProp<Request>,
   res: Response
 ): Promise<Response<any, Record<string, any>>> => {
-  // check is user has permissions
-  // check if person has ability to update enrollment
-  const payload: any = {
-    status: 'deleted'
-  };
-
   try {
-    const result = await EnrollmentDataLayer.updateById(
-      req.body.userId,
-      req.params.courseId,
-      payload
+    const result = await EnrollmentDataLayer.deleteById(
```

Comment:
We discussed this and decided on doing an update instead of a delete when a student is removed. we want to just switch their status to 'deleted' so that their progress is saved in case they get re-added.